virttest/utils_sys.py

```
@@ -74,6 +75,32 @@ def get_host_bridge_id(session=None):
+def is_image_mode(check_command="rpm-ostree", session=None):
+    """
+    Check if current OS is in image mode or package mode
+
+    :param check_command: str, the command to check the mode
+    :param session: aexpect session
+    :return: boolean, True for image mode, otherwise False
+    """
+    ret = True
+    if session:
+        status = session.cmd_status(check_command)
+        ret = False if status else True
```

Ruff (0.12.2)

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)